docs/cloud-integration.md

```diff
@@ -125,12 +125,13 @@ consult the relevant documentation.
 ### Recommended settings for writing to object stores
 
 For object stores whose consistency model means that rename-based commits are safe
-use the `FileOutputCommitter` v2 algorithm for performance:
+use the `FileOutputCommitter` v2 algorithm for performance, v1 for safety
```

Comment:
You're missing some punctuation at the end of almost all paragraphs you're touching.

Reply:
will review and correct